arm/Microsoft.Authorization/policyDefinitions/.parameters/mg.parameters.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "<<namePrefix>>-mg-policyDef"
+        },
+        "displayName": {
+            "value": "[DisplayName] This policy definition is deployed at the management group scope"
+        },
+        "description": {
+            "value": "[Description] This policy definition is deployed at the management group scope"
+        },
+        "policyRule": {
+            "value": {
+                "if": {
+                    "allOf": [
+                        {
+                            "field": "type",
+                            "equals": "Microsoft.Resources/subscriptions"
+                        },
+                        {
+                            "field": "[concat('tags[', parameters('tagName'), ']')]",
+                            "exists": "false"
+                        }
+                    ]
+                },
+                "then": {
+                    "effect": "modify",
+                    "details": {
+                        "roleDefinitionIds": [
+                            "/providers/microsoft.authorization/roleDefinitions/4a9ae827-6dc8-4573-8ac7-8239d42aa03f"
+                        ],
+                        "operations": [
+                            {
+                                "operation": "add",
+                                "field": "[concat('tags[', parameters('tagName'), ']')]",
+                                "value": "[parameters('tagValue')]"
+                            }
+                        ]
+                    }
+                }
+            }
+        },
+        "parameters": {
+            "value": {
+                "tagName": {
+                    "type": "String",
+                    "metadata": {
+                        "displayName": "Tag Name",
+                        "description": "Name of the tag, such as 'environment'"
+                    }
+                },
+                "tagValue": {
+                    "type": "String",
+                    "metadata": {
+                        "displayName": "Tag Value",
+                        "description": "Value of the tag, such as 'production'"
+                    }
+                }
+            }
+        },
+        "metadata": {
+            "value": {
+                "category": "Security"
+            }
+        },
+        "managementGroupId": {
```

Comment:
Isn't this value automatically provided by the pipeline (given the scope)

Reply:
I've removed on the 'min' and kept on the normal one to demonstrate the use of the parameter. It is 'optional' and will use the current deployment scope (thanks to the new managementGroup() function)